index.html

```diff
@@ -294,7 +294,7 @@ <h2 class="header-primary-text">contact me</h2>
                 built or a project that
                 needs
                 coding. I’d love to help with it</p>
-            <form action="https://formspree.io/f/xjvjagad" class="contact-form" method="POST">
```

Comment:
Please remove all unused files and folders like `.vscode` or add them to gitignore.